src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
```

Comment:
There is no way to tell which vendors are supported by this implementation.
I assume that this covers mastercard and VISA but I should be able to tell from class name (or javadoc at worst)